### PR TITLE
Add batch script for signing a release

### PR DESCRIPTION
**Pull request by SailReal**

### CODE REVIEW

File: SharkCage/util/sign_sharkcage_release.bat [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+echo 1. You started the script with the 'Developer Command Prompt for VS 2017' 
+echo 2. You provided the path to your solution folder e.g.: C:\HTWG_shark_cage\SharkCage
+echo 3. You have the Yubikey with the certificate (including the private key) available.
+
+set path_to_solution=%1
+
+echo Now build the complete solution, when it is done press enter.
+pause >nul
+
+signtool sign /d "CageChooser" /tr http://timestamp.digicert.com /td sha256 /fd sha256 /sha1 ADBE74BD39789DD111815DE59C60D715143E4620 %path_to_solution%\CageChooser\obj\Release\CageChooser.exe
```
Comment (DonatJR) [inline SharkCage/util/sign_sharkcage_release.bat:14]: The correct output path is: `%path_to_solution%\CageChooser\`__`bin`__`\Release\CageChooser.exe`
image

Not sure why the binary only copied to the `obj` folder.

File: SharkCage/util/sign_sharkcage_release.bat [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+signtool sign /d "CageConfigurator" /tr http://timestamp.digicert.com /td sha256 /fd sha256 /sha1 ADBE74BD39789DD111815DE59C60D715143E4620 %path_to_solution%\CageConfigurator\obj\Release\CageConfigurator.exe
```
Comment (DonatJR) [inline SharkCage/util/sign_sharkcage_release.bat:15]: same as above

**Review by DonatJR — CHANGES_REQUESTED** [review]
nice, this should make thinks easier when doing a release :)

File: SharkCage/util/sign_sharkcage_release.bat (hunk 1 above)
Comment (SailReal) [reply SharkCage/util/sign_sharkcage_release.bat:14]: The binary is first copied to the `obj` folder and after that to the `bin`. That was the problem, why the `Chooser` and the `Configurator` wasn't singed on the last "preprerelease". The output in the bin folder is replaced by the content of the obj. Found after searching a round an issue for that but I'd have to look again to find the link...

File: SharkCage/util/sign_sharkcage_release.bat (hunk 2 above)
Comment (SailReal) [reply SharkCage/util/sign_sharkcage_release.bat:15]: See https://github.com/SharkCagey/HTWG_shark_cage/pull/116#discussion_r211105957

File: SharkCage/util/sign_sharkcage_release.bat (hunk 1 above)
Comment (SailReal) [reply SharkCage/util/sign_sharkcage_release.bat:14]: This was the link: https://stackoverflow.com/a/22567659/825532

Comment (SailReal) [reply SharkCage/util/sign_sharkcage_release.bat:14]: And found no way to configure that the installer should use the `bin` folder instead of the `obj`

Comment (DonatJR) [reply SharkCage/util/sign_sharkcage_release.bat:14]: Ah, okay 👍  sorry for the confusion 😅

**Review by DonatJR — APPROVED** [review]

**Pull request merged**